libespm/src/Utils.cpp

```diff
@@ -42,7 +42,7 @@ bool IsItem(Type type) noexcept
 {
   return type == "AMMO" || type == "ARMO" || type == "BOOK" ||
     type == "INGR" || type == "ALCH" || type == "SCRL" || type == "SLGM" ||
-    type == "WEAP" || type == "MISC";
+    type == "WEAP" || type == "MISC" || type == "LIGH";
```

Comment:
let's add a comment why we add LIGH here

because in general light is not always an item.